docs/build-docs.sh

```diff
@@ -20,6 +20,7 @@ conda env create \
     --name docs-env \
     --file env.yml || exit 1
 
+# shellcheck source=/dev/null
```

Comment:
```suggestion
# shellcheck source=/dev/null
```

I can tell from running `shellcheck` locally that this is intended to address this `shellcheck` error:

```text
^------^ SC1091 (info): Not following: activate was not specified as input (see shellcheck -x).
```

Can you please explain how `shellcheck source=/dev/null` works (hopefully with a link to documentation)? I'm not familiar with that that does.

Reply:
I read the error description on shellcheck https://www.shellcheck.net/wiki/SC1091 and my understanding is that a `source` command will point to a script file to executed at that location. Shellcheck would like to parse that file too to proof read it, the `#shellcheck source=PATH_TO_SCRIPT` is supposed to point to the path of the file and allow a static tool to ingest the script. Unfortunately here the file does not exist until your run L17 creating that `docs-env` environment, hence the check cannot happen. One way for silencing the error is to point to `/dev/null` which can be assumed to be empty and close the check. A global exception to the `1091` rule could be added to a `.shellcheck` file but there may be places where the check could be and should be applied.

Reply:
```suggestion
# shellcheck disable=SC1091
```

Ah got it, ok thanks for that! Instead of doing this, let's please just skip this particular error on this particular line, as described in https://www.shellcheck.net/wiki/Ignore.

That makes it clear which specific check from `shellcheck` we're working around.